secure-device-storage/sds-android/securedevicestorage/src/main/java/de/adorsys/android/securedevicestorage/SecurePreferences.java

```diff
@@ -0,0 +1,111 @@
+package de.adorsys.android.securedevicestorage;
+
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
+import android.util.Log;
+
+import java.io.IOException;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.UnrecoverableEntryException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+
+import javax.crypto.NoSuchPaddingException;
+
+import static android.content.Context.MODE_PRIVATE;
+
+public class SecurePreferences {
+
+    private static final String KEY_SHARED_PREFERENCES_NAME = "SecurePreferences";
+
+    @RequiresApi(api = Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/pull/54/commits/825b5723dd621d327ac110cc7399f8372493cca4